Snakefile

```diff
@@ -32,14 +32,29 @@ rule fastq_dump:
         fastq-dump -O $(dirname {output.R1}) --split-files --gzip {wildcards.SRR_ID}
         """
 
+rule fastqc:
+    input:
+        R1 = int_dir / "fastqs" / "{SRR_ID}" / "{SRR_ID}_1.fastq.gz",
+        R2 = int_dir / "fastqs" / "{SRR_ID}" / "{SRR_ID}_2.fastq.gz",
+    output:
+        fastqc_1 = int_dir / "fastqc" / "{SRR_ID}" / "{SRR_ID}_1.fastq" /,
+        fastqc_2 = int_dir / "fastqc" / "{SRR_ID}" / "{SRR_ID}_2.fastq" /,
+    shell:
+        """
+        mkdir -p $(dirname {fastqc_1})
+        mkdir -p $(dirname {fastqc_2})
+        fastqc -o {output.fastqc_1} {input.R1}
+        fastqc -o {output.fastqc_2} {input.R2}
```

Comment:
I think the -o command takes a directory, not a file path so you'll need to do something like $(dirname {output.fastqc_1}) here.